fiftyone_devicedetection_examples/performance-tests/test_performance.py

```diff
@@ -80,31 +80,19 @@ def benchmark(data_file, user_agents_file):
     environment = {"detect": detect, "app": app, "pipeline": pipeline, "user_agents": user_agents}
     time = timeit("detect(app, pipeline, user_agents)", globals=environment, number=len(user_agents_list))
 
-    with open("performance_test_summary.json", "w") as summary:
+    if output:
         json.dump({
-            "HigherIsBetter": {
-                "Detections": len(user_agents_list),
-                "DetectionsPerSecond": len(user_agents_list) / time,
-            },
-            "LowerIsBetter": {
-                "RuntimeSeconds": time,
-                "AvgMillisecsPerDetection": time / len(user_agents_list) * 1000,
-            }
-        }, summary, indent=4)
-
-class DeviceDetectionPerformanceTests(unittest.TestCase):
-    def setUp(self):
-        self.data_file = "/home/user/Src/Work/assets/51Degrees-LiteV4.1.hash"
-        self.user_agents_file = "/home/user/Src/Work/assets/20000 User Agents.csv"
-        print("Data:", self.data_file)
-        print("User Agents:", self.user_agents_file)
-
-    def test_onpremise_device_detection_performance(self):
-        benchmark(self.data_file, self.user_agents_file)
+            "Detections": len(user_agents_list),
+            "DetectionsPerSecond": len(user_agents_list) / time,
+            "RuntimeSeconds": time,
+            "AvgMillisecsPerDetection": time / len(user_agents_list) * 1000,
+        }, sys.stdout, indent=4)
```

Comment:
This isn't going to work going to `stdout` right? There are other things also being written to `stdout` meaning the JSON is only part of the text.
We would need to pass the name of the JSON output file as a parameter rather than just a flag. This needs to be optional, so when running as an example, it operates as it does now, where no JSON is constructed

Reply:
Oh, for some reason I thought that by "outputting JSON data" you meant specifically outputting to stdout, so that it can be easily captured and formatted by the PowerShell script.

It works, because logs are being written to stderr, only the output goes to stdout.

Reply:
I see, thanks. Doesn't seem ideal to write to `stderr`, but it's always been like that I think, so let's not change it as part of this.

Reply:
Hi @ben51degrees do I understand correctly that no changes need here and this can be resolved?  Thanks

Reply:
Yes, I will resolve this,. Could you make raise an issue to look at why examples are writing to stderr rather than stdout? And I will get to that at some point.